infixevaluation/infixtopostfixandevaluation.py

```diff
@@ -0,0 +1,60 @@
+def infix_to_postfix(expression):
+    def precedence(operator):

```

Comment:
We do not encourage nested functions because the doctests in their docstrings never run.

If you really want nested functions then make sure their doctests are in the docstrings of the `infix_to_postfix()` function.